docs/components/radio.md

```diff
@@ -60,17 +64,76 @@ Will place label text to the right of the radio. Can also be [slotted](#slots).
 
 ### description
 
-Will place description text under the radio label (required). Can also be [slotted](#slots).
+Will place description text under the radio label. Can also be [slotted](#slots).
 
 ```html
 <KRadio :selected-value="true" v-model="radio" label="Label Example" description="Some subheader text" />
 ```
 
 <KRadio :selected-value="true" v-model="radio" label="Label Example" description="Some subheader text" />
 
-### selectedValue
+### type
 
-The value of the `KRadio` option that will be emitted by the `change` and `update:modelValue` events.
+Controls appearance of radio input element. Accepted values:
+- `card`
+
+::: warning NOTE
+`label` and `description` props, as well as `description` slot are ignored when `type` prop is `card`. You can only define content of a card via default slot.
+:::
+
+::: tip TIP
+You can choose to utilize the `.k-radio-label` and `.k-radio-description` classes within the slot as shown in the example below to leverage preconfigured styles.
+:::
+
+<KCard>
+  <template #body>
+    <div class="d-flex">
+      <KRadio type="card" selected-value="foo" v-model="cardRadio">
+        <img class="mb-2" src="/img/kong-logomark.png" alt="Kong logo" />
+        <div class="k-radio-label">Foo</div>
+        <div class="k-radio-description">This subheader</div>
+      </KRadio>
+      <KRadio type="card" selected-value="bar" v-model="cardRadio">
+        <img class="mb-2" src="/img/kong-logomark.png" alt="Kong logo" />
+        <div class="k-radio-label">Bar</div>
+        <div class="k-radio-description">That subheader</div>
+      </KRadio>
+    </div>
+    <div class="mt-3">Selected: {{ cardRadio }}</div>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KRadio type="card" selected-value="foo" v-model="cardRadio">
+    <img class="mb-2" src="/img/kong-logo.png" alt="Kong logo" />
+    <div class="k-radio-label">Foo</div>
+    <div class="k-radio-description">This subheader</div>
+  </KRadio>
+  <KRadio type="card" selected-value="bar" v-model="cardRadio">
+    <img class="mb-2" src="/img/kong-logo.png" alt="Kong logo" />
+    <div class="k-radio-label">Bar</div>
+    <div class="k-radio-description">That subheader</div>
+  </KRadio>
+  <div class="mt-3">Selected: {{ cardRadio }}</div>
+</template>
+
+<script lang="ts">
```

Comment:
I feel like it might be weird to have only example in vue 3 syntax while all other examples on this page in vue 2. Will add this to ticket about updating examples in docs.